action.yml

```diff
@@ -78,11 +78,29 @@ inputs:
   ispwConfigPath :
     description: 'The path to Code Pipeline YAML configuration'
     required: false
+  gitCommit :
+    description: 'The Git commit hash (long or short) or a colon-delimited list of file paths in the workspace'
```

Comment:
'or a colon-delimited list of file paths in the workspace.' this statement is already a part of 'gitCommitFile' parameter. I suggest to remove it from here unless it has some other significance.